src/main/java/teammates/ui/controller/InstructorFeedbackEditPageData.java

```diff
@@ -12,68 +13,400 @@
+        additionalSettings.setSessionVisibleDateValue(hasSessionVisibleDate ? 
+                                                      TimeHelper.formatDate(newFeedbackSession.sessionVisibleFromTime) :
+                                                      "");
+        additionalSettings.setSessionVisibleDateDisabled(!hasSessionVisibleDate);
+        
+        date = hasSessionVisibleDate ? newFeedbackSession.sessionVisibleFromTime : null;   
+        additionalSettings.setSessionVisibleTimeOptions(getTimeOptionsAsElementTags(date));
+        
+        additionalSettings.setSessionVisibleAtOpenChecked(Const.TIME_REPRESENTS_FOLLOW_OPENING.equals(
+                                                                   newFeedbackSession.sessionVisibleFromTime));
+        
+        additionalSettings.setSessionVisiblePrivateChecked(Const.TIME_REPRESENTS_NEVER.equals(
+                                                               newFeedbackSession.sessionVisibleFromTime));
+                        
+        boolean hasResultVisibleDate = !TimeHelper.isSpecialTime(newFeedbackSession.resultsVisibleFromTime);
+        
+        additionalSettings.setResponseVisibleDateChecked(hasResultVisibleDate);
+        
+        additionalSettings.setResponseVisibleDateValue(hasResultVisibleDate ? 
+                                                       TimeHelper.formatDate(newFeedbackSession.resultsVisibleFromTime) :
+                                                       "");
+        
+        additionalSettings.setResponseVisibleDateDisabled(!hasResultVisibleDate);
+        
+        date = hasResultVisibleDate ? newFeedbackSession.resultsVisibleFromTime :  null;
+        additionalSettings.setResponseVisibleTimeOptions(getTimeOptionsAsElementTags(date));
+        
+        additionalSettings.setResponseVisibleImmediatelyChecked(Const.TIME_REPRESENTS_FOLLOW_VISIBLE.equals(newFeedbackSession.resultsVisibleFromTime));
+        
+        additionalSettings.setResponseVisiblePublishManuallyChecked(
+                                 (Const.TIME_REPRESENTS_LATER.equals(newFeedbackSession.resultsVisibleFromTime) 
+                                  || Const.TIME_REPRESENTS_NOW.equals(newFeedbackSession.resultsVisibleFromTime)));
+        
+        additionalSettings.setResponseVisibleNeverChecked(Const.TIME_REPRESENTS_NEVER.equals(newFeedbackSession.resultsVisibleFromTime));
+        
+        additionalSettings.setSendClosingEmailChecked(newFeedbackSession.isClosingEmailEnabled);
+        additionalSettings.setSendOpeningEmailChecked(newFeedbackSession.isOpeningEmailEnabled);
+        additionalSettings.setSendPublishedEmailChecked(newFeedbackSession.isPublishedEmailEnabled);
+        
+        return additionalSettings;
+    }
+    
+    
+    
+    
+    public FeedbackSessionsForm getFsForm() {
+        return fsForm;
+    }
+
+
+    public List<FeedbackQuestionEditForm> getQnForms() {
+        return qnForms;
+    }
+
+    
+
```

Comment:
Keep the number of blank lines between methods consistent unless there's a proper reason for it.